### PR TITLE
Add monthly blog post about maplibre-rs

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: content/news/2022-06-26-maplibre-rs-monthly.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+title: "maplibre-rs monthly"
+date: 2022-06-26
+categories: ["maplibre-rs"]
+authors: [maxammann]
+draft: false
+---
+
+[maplibre-rs](https://github.com/maplibre/maplibre-rs) is an upcoming and cross-platform vector map renderer written in Rust. If you want to get in touch with the developers visit the [#maplibre:matrix.org](https://matrix.to/#/#mapr:matrix.org) chat or join the MapLibre (#maplibre and #maplibre-rs) community within the OpenStreetMap Slack. You can get an invite [here](https://osmus-slack.herokuapp.com/).
+
+asdfsdf
```
Comment (nyurik) [inline content/news/2022-06-26-maplibre-rs-monthly.md:11]: ??

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,5 @@
 {
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+// "spellright.language": ["en_US"],
```
Comment (nyurik) [inline .vscode/settings.json:3]: Not legal json

Comment (maxammann) [reply .vscode/settings.json:3]: Its legal and normal in vscode though :D blame microsoft :D

**Review by nyurik — APPROVED** [review]
lgtm, thx!

**Pull request merged**